src/data/move_animations.asm

```diff
@@ -0,0 +1,437 @@
+MoveAnimation_52c6: ; (6:52c6)
```

Comment:
Hope you had a script to help you, this is a lot of great stuff!

Reply:
Yeah.
Without my python friend, I'd copy and paste all the time :D